src/main/java/com/google/maps/GeocodingApiRequest.java

```diff
@@ -120,7 +120,7 @@ public GeocodingApiRequest region(String region) {
    * fully restrict the results from the geocoder.
    *
    * <p>For more information see <a
-   * href="https://developers.google.com/maps/documentation/geocoding/?hl=pl#ComponentFiltering">
+   * href="https://developers.google.com/maps/documentation/geocoding/intro?hl=pl#ComponentFiltering">
```

Comment:
Similar to my previous comment, the URL should be:
https://developers.google.com/maps/documentation/geocoding/intro#ComponentFiltering